juju/controller.py

```diff
@@ -257,13 +263,20 @@ def logout(self, force=False):
         """
         pass
 
-    def get_model(self, name):
+    async def get_model(self, uuid):
         """Get a model by name.
 
         :param str name: Model name
 
         """
-        pass
+        model_facade = client.ModelManagerFacade()
+        model_facade.connect(self.connection)
+        results = await model_facade.ModelInfo([client.Entity(tag.model(uuid))])
+        if results and results.results and results.results[0].result:
```

Comment:
We really need a better general pattern for digging through results like this. Not a complaint about this particular PR, though :-)